lib/lazy_high_charts/layout_helper.rb

```diff
@@ -81,8 +81,8 @@ def request_is_referrer?
       defined?(request) && request.respond_to?(:headers) && request.headers["X-XHR-Referer"]
     end
 
-    def request_turbolinks_5_tureferrer?
-      defined?(request) && request.respond_to?(:headers) && request.headers["Turbolinks-Referrer"]
+    def is_turbolinks_5?
+      Gem::Version.new(Turbolinks::VERSION) >= Gem::Version.new('5.0.0')
```

Comment:
maybe a mix of the two solutions ?
 ` is_turbolinks_5? && request_turbolinks_5_tureferrer?`